tests/test_fits.py

```diff
@@ -93,9 +86,9 @@ def test_write_exception(tmp_path):
 
 
 @pytest.mark.skipif(not HAVE_FITSIO, reason="test requires fitsio")
-def test_out_filename():
+def test_out_filename(tmp_path):
     import fitsio
 
-    fits = fitsio.FITS(filename, "rw", clobber=True)
+    fits = fitsio.FITS(tmp_path / filename, "rw", clobber=True)
     writer = user._FitsWriter(fits)  # noqa: SLF001
-    assert writer.fits._filename == filename  # noqa: SLF001
+    assert writer.fits._filename == f"{tmp_path}/{filename}"  # noqa: SLF001
```

Comment:
What is this actually testing? That `fitsio` works? In which case, none of our business here.

Reply:
Actually yes, the test is redundant.